examples/va/nfv/nfv-ovs-dpdk-sriov-hci/values.yaml

```diff
@@ -0,0 +1,23 @@
+# local-config: referenced, but not emitted by kustomize
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: edpm-nodeset-values-post-ceph
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  nodeset:
+    services:
```

Comment:
These services are duplicated in pre ceph steps also. can we remove these services in pre ceph and verify the services in post ceph.